src/main/java/org/ozwillo/dcexporter/model/DcModelMapping.java

```diff
@@ -174,6 +180,14 @@ public void setVersion(String version) {
         this.version = version;
     }
 
+    public List<String> getExcludedFields() {
+        return excludedFields;
+    }
+
+    public void setExcludedFields(List<String> excludedFields) {
```

Comment:
You can remove it also (and probably main of other setters in this class but that's another thing)